tests/test_wave_03.py

```diff
@@ -57,9 +57,10 @@ def test_friends_unique_movies_not_duplicated():
 
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
+    assert INTRIGUE_3 in friends_unique_movies
```

Comment:
In tests I recommend being as specific as possible to ensure that the function being tested is doing exactly what is expected.
```suggestion
    assert INTRIGUE_3 in friends_unique_movies
    assert HORROR_1 in friends_unique_movies
    assert FANTASY_4 in friends_unique_movies
```